Bug: helmfile lint error "apiVersion is required" when loading yaml manifest from git

**Problem**: 
Encounter the following error when trying to run `helmfile lint` when using yaml manifest from git

```
running helm template --debug=false --include-crds --output-dir /tmp/chartify714366862/helmx.1.rendered --namespace local prometheus-crds /tmp/chartify714366862
Linting release=prometheus-crds, chart=/tmp/chartify714366862
==> Linting /tmp/chartify714366862
[ERROR] Chart.yaml: apiVersion is required. The value must be either "v1" or "v2"
[INFO] Chart.yaml: icon is recommended
[INFO] values.yaml: file does not exist


in helmfile.d/00-crd.yaml: command "/home/linuxbrew/.linuxbrew/bin/helm" exited with non-zero status:

PATH:
  /home/linuxbrew/.linuxbrew/bin/helm

ARGS:
  0: helm (4 bytes)
  1: lint (4 bytes)
  2: /tmp/chartify714366862 (22 bytes)
  3: --namespace (11 bytes)
  4: local (5 bytes)

ERROR:
  exit status 1

EXIT STATUS
  1

STDERR:
  Error: 1 chart(s) linted, 1 chart(s) failed

COMBINED OUTPUT:
  Error: 1 chart(s) linted, 1 chart(s) failed
  ==> Linting /tmp/chartify714366862
  [ERROR] Chart.yaml: apiVersion is required. The value must be either "v1" or "v2"
  [INFO] Chart.yaml: icon is recommended
  [INFO] values.yaml: file does not exist
```

**Version:**
1. helmfile - `v0.130.1`
2. helm - `v3.2.1`

**helmfile**:
```yaml
helmDefaults:
  wait: true
  timeout: 1000
  createNamespace: true
releases:
  - name: prometheus-crds
    namespace: local
    # "8.13.8"
    chart: git::http://github.com/prometheus-operator/prometheus-operator.git@/example/prometheus-operator-crd?ref=release-0.38
```

**Command**: `helmfile lint`

**Expected Behavior**: helmfile lint run with success

**Extra**: Looks like when we are generating dynamic helm chart for the crd we are not adding `apiVersion` information in `Chart.yaml`. I also try to debug the problem and find following content in the generated `Chart.yaml`

```yaml
name: "prometheus-operator-crd"
version: 1.0.0
appVersion: 1.0.0
```

I will also trying to create MR but as I am new to code not able to find it till yet, will also try to create a MR for the fix.
Thanks :)

--- a/go.mod
+++ b/go.mod
@@ -22,7 +22,7 @@ require (
 	github.com/spf13/cobra v0.0.3
 	github.com/tatsushid/go-prettytable v0.0.0-20141013043238-ed2d14c29939
 	github.com/urfave/cli v1.20.0
-	github.com/variantdev/chartify v0.4.2
+	github.com/variantdev/chartify v0.4.3
 	github.com/variantdev/dag v0.0.0-20191028002400-bb0b3c785363
 	github.com/variantdev/vals v0.10.2
 	go.uber.org/multierr v1.1.0

